analysis/deep2_r23_o32/plotting/color_bin_plotting.py

```diff
@@ -10,14 +10,15 @@
 
 def color_for_bin(fitspath, bin_info, pdf_file):
     """
-    Purpose
-    Plots for R23 and O32 for Voronoi analysis for each bin
+    Purpose:
+      Plots for R23 and O32 for Voronoi analysis for each bin
 
-    Parameters
-    fitspath -> path where files are called from and saved to
-    bin_info -> table created by binning code
-    pdf_file -> name of pdf file produced
+    Parameters:
+      fitspath -> path where files are called from and saved to
+      bin_info -> table created by binning code
+      pdf_file -> name of pdf file produced
```

Comment:
Parameters
-----------
param: fitspath: str. path where files are called from and saved to
param: bin_info: str. table created by binning code
param: pdf_file: str. name of pdf file produced

Reply:
I created #64 for this issue. Will address outside this PR.